lib/task/pdf/ead-pdf-common.xsl

```diff
@@ -0,0 +1,1259 @@
+            <xsl:call-template name="toc"/>
+            <fo:block xsl:use-attribute-sets="smp">
+                <fo:list-block>
+                    <xsl:for-each select="ead:odd[starts-with(@type, 'title')]">
+                        <xsl:call-template name="titleNoteListItem"/>
+                    </xsl:for-each>
+                </fo:list-block>
+            </fo:block>
+        </xsl:if>
+    </xsl:template>
+    <xsl:template name="titleNoteListItem">
+        <fo:list-item>
+            <fo:list-item-label end-indent="label-end()">
+                <fo:block>•</fo:block>
+            </fo:list-item-label>
+            <fo:list-item-body start-indent="body-start()">
+                <fo:block>
+                    <fo:inline text-decoration="underline" font-weight="bold">
+                        <xsl:value-of select="local:titleNoteLabel(.)"/>
+                    </fo:inline>: <xsl:value-of select="."/>
+                </fo:block>
+            </fo:list-item-body>
+        </fo:list-item>
+    </xsl:template>
+    <!-- Named template to link back to the table of contents -->
+    <xsl:template name="toc">
+        <!-- Uncomment to enable 'back to toc' links on each section.
+        <fo:block font-size="11pt" margin-top="12pt" margin-bottom="18pt">
+            <fo:basic-link text-decoration="none" internal-destination="toc" color="#14A6DC">
+            <fo:inline font-weight="bold">^</fo:inline> Return to Table of Contents </fo:basic-link>
+        </fo:block>
+        -->
+    </xsl:template>
+    <!-- Cover page templates -->
+    <!-- Builds title -->
+    <xsl:template match="ead:titlestmt" mode="pageHeader">
+        <xsl:apply-templates select="ead:titleproper[1]"/>
+    </xsl:template>
+    <xsl:template match="ead:titlestmt" mode="coverPage">
+        <!-- Calls template with links to archive logo -->
+        <fo:block border-bottom="1pt solid #666" margin-top="1.5cm" id="cover-page">
+            <xsl:call-template name="logo"/>
+            <fo:block xsl:use-attribute-sets="h1">
+                Finding Aid -
+                <xsl:apply-templates select="ead:titleproper[1]"/>
+                (<xsl:value-of select="//ead:eadid"/>)
+                <xsl:if test="ead:subtitle">
+                    <fo:block font-size="16" font-weight="bold">
+                        <xsl:apply-templates select="ead:subtitle"/>
+                    </fo:block>
+                </xsl:if>
+            </fo:block>
+        </fo:block>
+        <fo:block margin-top="8pt">
+            <xsl:apply-templates select="/ead:ead/ead:eadheader/ead:profiledesc"/>
+        </fo:block>
+        <fo:block margin-top="8pt">
+            <xsl:apply-templates select="/ead:ead/ead:eadheader/ead:filedesc/ead:editionstmt"/>
+        </fo:block>
+    </xsl:template>
+    <xsl:template match="ead:publicationstmt" mode="coverPage">
+        <fo:block margin="0 0.3in">
+            <!-- Atom:
```

Comment:
Is this commented out code for a comment?

Reply:
Hmm, uncommenting that line adds a duplicate "publisher" (e.g. repository) name in the address block of the cover page:

![Screenshot 2022-06-17 081411](https://user-images.githubusercontent.com/626135/174327172-f25ac701-a469-4502-ab94-76bb8795ed64.png)

I don't understand how the publisher name is printed *without* that commented line, but it's working so I'll remove the unnecessary code and comment.